client/src/pages/auth/sign-in.tsx

```diff
@@ -32,9 +33,21 @@ const SignIn: React.FC = () => {
   const handleSignIn = useCallback(
     (data) => {
       setIsLoading(true);
-      signIn('credentials', { ...data, callbackUrl: query.callbackUrl || '/analysis' });
+      signIn('credentials', {
+        ...data,
+        redirect: false,
+      }).then(({ error }) => {
+        if (error) {
+          setIsLoading(false);
+          setError('password', {
+            message: 'login failed. incorrect password',
+          });
```

Comment:
Here you are assuming the error is because the password is wrong. We don't really know only with this piece of code. Anyway, the best, in this case, is to show a notification error as we have here: https://github.com/Vizzuality/landgriffon/blob/staging/client/src/containers/scenarios/component.tsx#L56